octavia-cli/octavia_cli/entrypoint.py

```diff
@@ -79,9 +101,10 @@ def octavia(ctx: click.Context, airbyte_url: str, workspace_id: str, enable_tele
         click.echo(click.style("🐙 - Project is not yet initialized.", fg="red", bold=True))
 
 
-def get_api_client(airbyte_url):
+def get_api_client(airbyte_url: str, header_name: Optional[str] = None, header_value: Optional[str] = None):
     client_configuration = airbyte_api_client.Configuration(host=f"{airbyte_url}/api")
-    api_client = airbyte_api_client.ApiClient(client_configuration)
+    api_client = airbyte_api_client.ApiClient(client_configuration, header_name=header_name,
+                                              header_value=header_value)
```

Comment:
I'd rather use the `set_default_header` method of the `ApiClient`. Especially if you agree to allow users to pass multiple headers name/values.